src/main/java/datawave/microservice/authorization/preauth/ProxiedEntityX509Filter.java

```diff
@@ -127,7 +128,7 @@ protected boolean principalChanged(HttpServletRequest request, Authentication cu
     }
     
     protected List<SubjectIssuerDNPair> getSubjectIssuerDNPairs(String proxiedSubjects, String proxiedIssuers) {
-        if (StringUtils.isEmpty(proxiedSubjects)) {
+        if (ObjectUtils.isEmpty(proxiedSubjects)) {
```

Comment:
Using this method has overhead as it has to do isInstanceOf checks.  I would prefer to use !StringUtils.hasLength(proxiedSubjects)